docs/application/web/guides/connectivity/bluetooth.md

```diff
@@ -152,31 +279,31 @@ To use the Bluetooth functionality of the device, you must switch the Bluetooth
    tizen.application.launchAppControl(bluetoothVisibilityAppControl, null, null, launchVisibilityError, serviceVisibilityReply);
    ```
 
-6. Set a friendly name for the device using the `setName()` method.
-
-   The name helps to recognize the device in a list of [retrieved devices](#discovering-bluetooth-devices).
+6. Set a friendly name for the device using the `setName()` method. The name helps to recognize the device in a list of [retrieved devices](#discover-bluetooth-devices).
```

Comment:
For step 5 as well as step 6, please use a colon instead of a period as it is followed by a ciode:

6. Set a friendly name for the device using the `setName()` method. The name helps to recognize the device in a list of [retrieved devices](#discover-bluetooth-devices):

Reply:
Done